### ISSUE TITLE
Uncaught TypeError: element.addEventListener is not a function

### ISSUE THREAD

**Issue by Maelstrome26** [sindresorhus/atom-jshint#140]
Repro:

Open a Javascript file

___

**Atom Version**: 1.1.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: [jshint](https://github.com/sindresorhus/atom-jshint) package, v1.8.1

### Stack Trace

Uncaught TypeError: element.addEventListener is not a function

```
At C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\src\delegated-listener.js:36

TypeError: element.addEventListener is not a function
  at listen (C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\src\delegated-listener.js:36:11)
  at [object Object].Tooltip.init (C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\src\tooltip.js:83:28)
  at new Tooltip (C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\src\tooltip.js:18:8)
  at TooltipManager.module.exports.TooltipManager.add (C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\src\tooltip-manager.js:50:17)
  at addReasons (C:\Users\Admin\.atom\packages\jshint\index.js:186:30)
  at displayError (C:\Users\Admin\.atom\packages\jshint\index.js:206:2)
  at C:\Users\Admin\.atom\packages\jshint\index.js:214:10
  at arrayEach (C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\lodash\index.js:1289:13)
  at Function.<anonymous> (C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\lodash\index.js:3345:13)
  at displayErrors (C:\Users\Admin\.atom\packages\jshint\index.js:213:4)
  at complete (C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\lodash\index.js:7697:25)
  at delayed (C:\Users\Admin\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\babel-core\node_modules\lodash\index.js:7707:11)

```

### Commands

```
     -1:45 editor:select-line (atom-text-editor.editor)
     -1:37.6.0 settings-view:open (atom-text-editor.editor.is-focused)
     -1:17 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-base16-tomorrow-dark-theme.theme-one-dark-ui)
     -0:26.3.0 command-palette:toggle (div.panels)
     -0:24.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:24.4.0 sync-settings:backup (div.panels)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "language-objective-c",
      "language-c",
      "language-clojure",
      "language-coffee-script",
      "language-csharp",
      "language-go",
      "language-java",
      "language-mustache",
      "language-perl",
      "language-ruby",
      "language-ruby-on-rails",
      "language-toml",
      "autocomplete-paths",
      "color-picker",
      "editor-stats"
    ],
    "themes": [
      "one-dark-ui",
      "base16-tomorrow-dark-theme"
    ]
  }
}
```

### Installed Packages

```coffee
# User
aligner, v0.17.1
aligner-css, v1.2.0
aligner-php, v1.0.0
atom-color-highlight, v3.0.8
autoclose-html, v0.19.0
css-color-underline, v1.0.1
css-snippets, v0.9.0
docblockr, v0.7.3
file-icons, v1.6.11
highlight-line, v0.11.0
highlight-selected, v0.10.1
indent-guide-improved, v1.4.3
jquery-snippets, v11.0.0
jshint, v1.8.1
line-ending-converter, v1.3.2
linter, v1.10.0
linter-jsonlint, v1.0.1
linter-php, v1.0.5
minimap, v4.15.0
minimap-pigments, v0.1.7
php-twig, v4.0.0
pigments, v0.18.1
pretty-json, v0.4.1
sync-settings, v0.6.0
tab-control, v0.6.8
trailing-spaces, v0.3.2
tree-view-git-status, v0.2.2

# Dev
No dev packages
```

**Comment by FabianMeul** [comment]
Also having this issue after updating to 1.1.0.

**Comment by rohanchandane** [comment]
+1

**Comment by davepies** [comment]
+ 1

**Comment by wjdhamilton** [comment]
+1

**Comment by AlexandreAWE** [comment]
+1

**Comment by joeyfromspace** [comment]
+1

Here's the .jshintrc I'm using:

{
  "bitwise": true,
  "camelcase": true,
  "curly": false,  /* custom */
  "eqeqeq": true,
  "es3": true,
  "forin": true,
  "freeze": true,
  "immed": true,
  "indent": 1,  /* custom */
  "latedef": true,
  "newcap": true,
  "noarg": true,
  "noempty": true,
  "nonbsp": true,
  "nonew": true,
  "plusplus": false,  /* custom */
  "quotmark": true,
  "undef": true,
  "unused": true,
  "strict": false, /* enable later */
  "trailing": true,
  "maxparams": false,
  "maxdepth": false,
  "maxstatements": false,
  "maxcomplexity": false,
  "maxlen": false,
  "asi": false,
  "boss": true,  /* custom */
  "debug": false,
  "eqnull": true,  /* custom */
  "esnext": false,
  "evil": false,
  "expr": false,
  "funcscope": false,
  "gcl": false,
  "globalstrict": false,
  "iterator": false,
  "lastsemic": false,
  "laxbreak": false,
  "laxcomma": false,
  "loopfunc": false,
  "moz": false,
  "multistr": false,
  "notypeof": false,
  "proto": false,
  "scripturl": false,
  "smarttabs": false,
  "shadow": false,
  "sub": false,
  "supernew": false,
  "noyield": false,
  "browser": true,  /* custom */
  "node": true,  /* custom */
  "onevar": true,  /* custom */
  "globals": {  /* custom */
    /* MOCHA: http://stackoverflow.com/a/19572515/130638 */
    "describe"   : false,
    "it"         : false,
    "before"     : false,
    "beforeEach" : false,
    "after"      : false,
    "afterEach"  : false
  }
}

**Atom Version**: 1.1.0
**System**: Mac OS X 10.11.1
**Thrown From**: [jshint](https://github.com/sindresorhus/atom-jshint) package, v1.8.1


### Stack Trace

Uncaught TypeError: element.addEventListener is not a function

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/delegated-listener.js:36

TypeError: element.addEventListener is not a function
    at listen (/Applications/Atom.app/Contents/Resources/app.asar/src/delegated-listener.js:36:11)
    at [object Object].Tooltip.init (/Applications/Atom.app/Contents/Resources/app.asar/src/tooltip.js:83:28)
    at new Tooltip (/Applications/Atom.app/Contents/Resources/app.asar/src/tooltip.js:18:8)
    at TooltipManager.module.exports.TooltipManager.add (/Applications/Atom.app/Contents/Resources/app.asar/src/tooltip-manager.js:50:17)
    at addReasons (/Users/joey/.atom/packages/jshint/index.js:169:32)
    at displayError (/Users/joey/.atom/packages/jshint/index.js:189:2)
    at /Users/joey/.atom/packages/jshint/index.js:196:24
    at arrayEach (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/babel-core/node_modules/lodash/index.js:1289:13)
    at Function.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/babel-core/node_modules/lodash/index.js:3345:13)
    at displayErrors (/Users/joey/.atom/packages/jshint/index.js:196:4)
```

### Commands

```
     -0:25.5.0 core:move-right (atom-text-editor.editor.mini.is-focused)
     -0:25.3.0 editor:select-to-beginning-of-word (atom-text-editor.editor.mini.is-focused)
     -0:24.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:23.7.0 core:close (atom-text-editor.editor.is-focused)
     -0:19 core:move-right (atom-text-editor.editor.is-focused)
     -0:18.7.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:18.4.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:18.1.0 core:copy (atom-text-editor.editor.is-focused)
     -0:17.8.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:15 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:14.5.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:14.2.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:13.9.0 core:paste (atom-text-editor.editor.is-focused)
     -0:11.3.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:07.7.0 core:save (atom-text-editor.editor.is-focused)
     -0:00.5.0 editor:newline (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "jslint"
    ]
  }
}
```

### Installed Packages

```coffee
# User
jshint, v1.8.1

# Dev
No dev packages
```

**Comment by DiegoCardoso** [comment]
+1

**Comment by mrfelton** [comment]
wow, this makes atom completely unusable!

**Comment by bruinebeer** [comment]
+1

**Comment by dacodekid** [comment]
Tested few times and it happens only after Atom updates to 1.1.0. Works fine on Atom __1.0.19__

**Comment by rohanchandane** [comment]
Yeah as @joeyfromspace did, disable jslint for sometime, till it gets fixed with 1.1.0!

**Comment by jonjes** [comment]
+1

**Comment by sohamkamani** [comment]
+1

**Comment by Tenkaklet** [comment]
+1

**Comment by jkrenge** [comment]
+1

**Comment by schricka** [comment]
+1

**Comment by arthurvr** [comment]
Duplicate of https://github.com/sindresorhus/atom-jshint/issues/124.

**Comment by sindresorhus** [comment]
This was fixed in 1.8.2.
